public/javascripts/now.js

```diff
@@ -559,11 +561,11 @@ function updateMetadata() {
   if (nowPlaying()) {
     document.title = '"' + title + '" by ' + artist;
     if (cookieEnabled('extendedOn')) {
-      toggleDisplay('#userLine', true);
+      toggleDisplay(resources.state.extendedInfo, true);
     }
   } else {
     document.title = 'Last.fm Now';
-    toggleDisplay('#userLine', false);
+    toggleDisplay(resources.state.extendedInfo, false);
```

Comment:
I realized making this a part of the extended info will cause it to disappear when nothing is playing. Could you make it a separate hotkey like `t`? Remember to add it to the help menu and create a cookie for it.